libraries/core/src/main/kotlin/com/bumble/appyx/core/node/ParentNode.kt

```diff
@@ -89,6 +91,19 @@ abstract class ParentNode<NavTarget : Any>(
         manageTransitions()
     }
 
+    fun onPortalRemoved(node: Node) {
+        parent?.onPortalRemoved(this)
+        childNodeLifecycleManager.removePortal()
```

Comment:
Double check if the assumption that we can remove the last one w/o a reference will always be correct